src/Tickets/Commerce/Gateways/PayPal/Webhooks/Webhook_Register.php

```diff
@@ -21,33 +21,33 @@ class WebhookRegister {
 	 *
 	 * @var string[]
```

Comment:
change docblock where it says `Structure: PayPalEventName => EventHandlerClass` to `Structure: PayPal_Event_Name => Event_Handler_Class`